app/code/Magento/Captcha/Model/DefaultModel.php

```diff
@@ -517,7 +524,7 @@ protected function _randomSize()
      * Now deleting old captcha images make crontab script
      * @see \Magento\Captcha\Cron\DeleteExpiredImages::execute
      */
-    protected function _gc()
+    protected function gc()
```

Comment:
Please, add SuppressWarnings here. Something like that:

```
Added SuppressWarnings since this method is declared in parent class and we can not use other method name.
@SuppressWarnings(PHPMD.ShortMethodName)
```